src/utils/timezones.ts

```diff
@@ -0,0 +1,471 @@
+// List of timezones from iana database
+const timeZones = [
```

Comment:
I believe it already exists in stripes-component or somwhere in stripes-*

Reply:
I don't think it exists.

Reply:
Ok, Noah just added it to [stripes-components](https://github.com/folio-org/stripes-components/pull/2225) and [stripes-types](https://github.com/folio-org/stripes-types/pull/44). I have updated the usage in this module as well.